libs/openFrameworksCompiled/project/makefileCommon/config.shared.mk

```diff
@@ -70,32 +70,39 @@ endif
 
 
 ifdef MAKEFILE_DEBUG
-    $(info PLATFORM_ARCH=$(PLATFORM_ARCH))
-    $(info PLATFORM_OS=$(PLATFORM_OS))
-    $(info HOST_ARCH=$(HOST_ARCH))
-    $(info HOST_OS=$(HOST_OS))
-    $(info CROSS_COMPILING=$(CROSS_COMPILING))
-    $(info PLATFORM_VARIANT=$(PLATFORM_VARIANT))
-    $(info IS_RASPBIAN=$(IS_RASPBIAN))
+	$(info PLATFORM_ARCH=$(PLATFORM_ARCH))
+	$(info PLATFORM_OS=$(PLATFORM_OS))
+	$(info HOST_ARCH=$(HOST_ARCH))
+	$(info HOST_OS=$(HOST_OS))
+	$(info CROSS_COMPILING=$(CROSS_COMPILING))
+	$(info PLATFORM_VARIANT=$(PLATFORM_VARIANT))
+	$(info IS_RASPBIAN=$(IS_RASPBIAN))
 endif
 
 # if not defined, construct the default PLATFORM_LIB_SUBPATH
 ifndef PLATFORM_LIB_SUBPATH
 	# determine from the arch
 	ifeq ($(PLATFORM_OS),Linux)
 		ifeq ($(PLATFORM_ARCH),x86_64)
-			PLATFORM_LIB_SUBPATH=linux64
+			PLATFORM_LIB_SUBPATH=linux/64
```

Comment:
looks like older projects with addons in lib/linux64/ might not work based on this?